### DISCUSSION TITLE
Request - LXC for Bookstack - We use for IT documentation and works with Diagrams.net!

### DISCUSSION THREAD

**Discussion by Alpha3Cloud** [tteck/Proxmox#3492] [Ideas]
[Bookstack](https://www.bookstackapp.com/) is a great opensource project that is gaining traction for IT documentation. It has built in support for Diagrams.net and is easily customized for multiple use cases.

**Comment by tteck** [comment]
BookStack is available via the TurnKey LXC Appliances script.
https://github.com/tteck/Proxmox/blob/daa896894dc796f3a8b5fa98469af3d369f2bb4c/turnkey/turnkey.sh#L69
